### ISSUE TITLE
Check TabBar and Slider components for accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#692]
#### Description / User Story

After #595 we should also check components again, that were not finished when the first check was done:
- TabBar
- SliderSingleValue
- SliderTwoValues

----
#### Requirements
- [ ] #501 
- [x] #551 
- [ ] #268 
- [x] #538 
- [x] #539

----
#### Acceptance Criteria
- [ ] TabBar component was checked and findings that need to be fixed were added to a follow up task
- [ ] SliderSingleValue component was checked and findings that need to be fixed were added to a follow up task
- [ ] SliderTwoValues component was checked and findings that need to be fixed were added to a follow up task

----
#### Background information
- ...
